assets/javascripts/kitten/components/carousel/simple-carousel.js

```diff
@@ -59,89 +113,56 @@ class SimpleCarouselBase extends Component {
 
     const { totalPagesCount, currentPageNumber } = this.state
     const rangePage = createRangeFromZeroTo(totalPagesCount)
-
-    const containerCustomStyle = [styles.container, containerStyle]
-
-    const paginationCustomStyle = [
-      styles.pagination,
-      paginationAlign && { justifyContent: paginationAlign },
-      paginationStyle,
-    ]
+    const id = this.props.id ? this.props.id + '_' : ''
 
     return (
       <Fragment>
-        <Marger
-          bottom={this.showPagination() ? 4 : 0}
-          style={containerCustomStyle}
+        <StyledContainer
+          style={containerStyle}
+          addBottomMargin={this.showPagination()}
         >
           {React.Children.map(children, (item, index) => {
-            const itemStyle = [
-              styles.item,
-              index !== currentPageNumber && styles.item.hide,
-            ]
-
             return (
-              <div key={item.key} style={itemStyle}>
+              <div
+                key={item.key}
+                aria-hidden={index !== currentPageNumber}
+                id={id + 'carouselItem_' + index}
```

Comment:
Pour de l'interpolation, on peut l'écrire `${id}carouselItem_${index}`. Du coup je trouve ca plus lisible d'ajouter le `_` ici à la place de le faire plus haut. Qu'en penses-tu ?

Reply:
Si `this.props.id` est vide je sais pas si l'id `_carouselItem_1` est optimal (mais je ne pense pas que ça pose des problèmes)

Reply:
Ouais c'est pas le plus opti. On peut se contenter de l'interpolation avec les back ticks du coup.